### ISSUE TITLE
Error: DRAFT release must not have fraction

### ISSUE THREAD

**Issue by tobiasschuerg** [r0adkll/upload-google-play#139]
**Describe the bug**
Seems like userFraction is automatically set to `1` if not set manually.

If the configuration below is used to create a release draft, there is an error message saying that userFraction must not bet set.

> Error: DRAFT release must not have fraction

**Workflow Step Configuration**

yml

```yml
- name: Deploy xxx Google Play
  uses: r0adkll/upload-google-play@master
  with:
    serviceAccountJsonPlainText: ${{secrets.SERVICE_ACCOUNT_JSON}}
    packageName: xxx
    releaseFiles:
      app/build/outputs/bundle/xxx/xxx.aab
    track: internal
    status: draft
    changesNotSentForReview: true
    mappingFile: app/build/outputs/mapping/xxx/mapping.txt
```

Not that ´userFraction´ is not set

this is what is logged

![image](https://user-images.githubusercontent.com/1324555/189591204-91a7bef8-31d1-437b-9f60-5c22d1f82dda.png)

**Comment by tobiasschuerg** [comment]
> ### userFraction
> **number**
> Fraction of users who are eligible for a staged release. 0 < fraction < 1. Can only be set when status is "inProgress" or "halted".

https://developers.google.com/android-publisher/api-ref/rest/v3/edits.tracks#release

**Comment by boswelja** [comment]
Thanks for reporting this! I will update the logic for this ASAP

**Comment by boswelja** [comment]
Fixed by #140 
Until a release is created, I recommend rolling back a version
